Private/Find-ESC5.ps1

```diff
@@ -70,7 +70,9 @@
         [Parameter(Mandatory)]
         $SafeUsers,
         [Parameter(Mandatory)]
-        $SafeObjectTypes
+        $SafeObjectTypes,
```

Comment:
Minor, but would be nice to get consistently strictly typed parameters throughout the functions. Not a stopper for this release, though.